.github/workflows/build_nightly.yml

```diff
@@ -139,6 +139,105 @@ jobs:
             _build/tests/cobol85/**/duration.txt
 
 
+  build_clang:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        isam:
+          - db
+          - visam
+
+    steps:
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup build environment
+        run: echo "TERM=vt100" >> $GITHUB_ENV
+
+      - name: Install packages
+        run: |
+          sudo apt -y update
+          sudo apt -y install autoconf automake make libtool flex bison \
+                              autopoint gettext help2man texinfo texlive \
+                              clang libgmp-dev libncurses-dev \
+                              libdb-dev libxml2-dev libcjson-dev
+
+      - name: Install VISAM
+        if: ${{ matrix.isam == 'visam' }}
+        run: |
+          curl -LO http://inglenet.ca/Products/GnuCOBOL/visam-2.2.tar.Z
+          tar -xvzf visam-2.2.tar.Z
+          cd visam-2.2
+          ./configure --prefix=/opt/visam --enable-vbisamdefault
+          make && sudo make install
```

Comment:
the result before "make install" should be archived - to only do this if the archive is not already cached;
also, as there is no `--enable-debug --enable-hardening` in V-ISAM yet, the relevant flags should be added directly and as that is a clang build use clang as well - that has the benefit that GnuCOBOL's calls into V-ISAM are hardened as well and debugging would be easier - similar for the db build (both configure options and caching)

`configure CFLAGS="-fstack-protector-strong -fstack-clash-protection -ggdb3 -fasynchronous-unwind-tables" LDFLAGS="-fstack-protector-strong -fstack-clash-protection" CPPFLAGS="-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2" `